lib/Controllers/create.js

```diff
@@ -17,7 +17,29 @@ Create.prototype.plurality = 'plural';
 Create.prototype.write = function(req, res, context) {
   context.attributes = _.extend(context.attributes, req.body);
 
-  var self = this;
+  var self = this,
```

Comment:
https://github.com/dchester/epilogue/blob/master/lib/Controllers/update.js#L32

this is code dupe at very least. If we can cache some of the data we should, otherwise perhaps this can be rolled into a reusable function in the BaseController?